synapse/handlers/relations.py

```diff
@@ -412,14 +409,18 @@ async def _get_threads_for_events(
                 if event is None:
                     continue
 
-                potential_events, _ = await self.get_relations_for_event(
-                    event_id,
-                    event,
-                    room_id,
-                    RelationTypes.THREAD,
-                    ignored_users,
+                # Attempt to find another event to use as the latest event.
+                potential_events, _ = await self._main_store.get_relations_for_event(
```

Comment:
I'm confused, haven't we deleted this function `get_relations_for_event`?

Reply:
We deleted the one in the handler, not the one in the store.

Reply:
OH ha